edx_proctoring/tests/test_api.py

```diff
@@ -103,6 +103,7 @@ def setUp(self):
         self.start_an_exam_msg = 'This exam is proctored'
         self.exam_expired_msg = 'The due date for this exam has passed'
         self.timed_exam_msg = '{exam_name} is a Timed Exam'
+        self.submitted_timed_exam_msg_with_due_date = 'Once the exam due date has passed,'
```

Comment:
we should avoid "once" - has multiple meanings.  
"After the exam due date has passed,"